markdown/articles/2020/03/22/DDFAQs.md

```diff
@@ -0,0 +1,600 @@
+, which you should really have a read of before going through this
+one, however it seems I didn't touch on many points so hopefully this
+should provide some clarity on the following things:
+
+### DDFAQs
+
+**1. The different Doomsday archetypes
+1.1 DDFT
+1.2 DDEFT
+1.3 Meandeck
+2. Card Choices
+**2.1 Doomsday
+2.2 Mana Acceleration and Fixing
+2.3 Cantrips
+2.4 Protection
+2.5 Combo Pieces
+2.6 Manabase
+2.7 Sideboard
+3. Colour Choices
+**3.1 White
+3.2 Red
+3.3 Green
+3.4 No Splash
+4. The Mechanics of Resolving Doomsday
+**5. Learning "Piles"
+6. Additional Content Available
+
+
+## 1. The Different Doomsday Archetypes
+
+Although it wasn't always the case, it seems currently the most 
+popular medium for discussing specific Legacy archetypes (at the 
+time of writing) is via dedicated Discord servers for each
+archetype. The Doomsday community is no exception to this and
+though it may seem like a very aggressive and somewhat shameless
+plug so early on in this article, there is a purpose to this.
+
+Until recently we had one primary discussion channel for Legacy
+Doomsday with no real regard for the different variants however with
+so many people joining the server for one specific variant or another, 
+we had to do a little tidy up. Part of this process included listing 
+the variants out so that people could read a summary of each and select
+the one/(s) they liked the sound of to have access to those select
+parts of the server. Despite these summaries being present we still get
+some confusion as to what the differences actually entail both in terms
+of deck building decisions and in terms of personal preferences or
+gameplay styles.
+
+I will present the three main archetypes below with the current Discord 
+summary and a bit more comparison on each. 
+
+
+
+### 1.1 DDFT (DoomsDay Fetchland Tendrils)
+
+Discord Summary: *Lists that utilise traditional elements such as
+Burning WIsh, some sort of "draw 3/4" engine and focus on a Storm
+style approach to winning (however do not always require storm to win).
+
+[As mentioned last time](/article/2020/03/12/pretty-mean-deck.md), DDFT
+is the traditional name of historical iterations of the deck. We won't
+do a full history lesson here however it basically refers to the fact
+that this is a storm deck that can play a much longer game if required.
+
+The deck normally incorporates some sort of raw card advntage engine
+such as {{Ideas Unbound}}, {{Act on Impulse}}, {{Three Wishes}} or 
+{{Cruel Bargain}} to enable drawing into the Doomsday pile. {{Burning Wish}}
+has been used often to enable additional pseudo copies of Doomsday and 
+newer cards like {{Echo of Eons}} ave been incorporated in lists too.
+The aim of the deck is to kill via a Doomsday pile with lethal
+{{Tendrils of Agony}} or {{Thassa's Oracle}} or a natural Tendrils
+storm kill without Doomsday. See an example pile below:
+
+<pile>{{!AOI}} {{!LED}} {{!ToA}} {{!PN}} {{!CB}}</pile>
```

Comment:
I think this pile is a little out of date, haven't been playing AoI for a bit. I don't really know what a better pile would be though. maybe [D4, LP, DR, DR, ToA] if you want to include ToA in it, or perhaps [IU LED LED Ponder BW]

Reply:
Replaced.